### PR TITLE
Update status-url

### PR DESCRIPTION
**Pull request by wangxiyuan**
update `status-url` to the specified place.

### CODE REVIEW

**Comment by liusheng** [comment]
looks good. tested ?

**Comment by wangxiyuan** [comment]
> looks good. tested ?

yeah, I have add it in all-in-one env: https://github.com/liusheng/hadoop/pull/1

File: zuul.d/pipelines.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@
       github:
         status: pending
         comment: false
-        status-url: https://status.openlabtesting.org/status
+        status-url: https://status.openlabtesting.org/status/change/{change.number},{change.patchset}
```
Comment (kiwik) [inline zuul.d/pipelines.yaml]: @Yikun have committed the changes in PR https://github.com/theopenlab/labkeeper/pull/163/files and have applied it in OpenLab CI zuul node.

@Yikun please help to review and make sure whether we need the updates, thank you

**Comment by Yikun** [comment]
I changed this starting link before, https://github.com/theopenlab/openlab-zuul-jobs/pull/581

LGTM, this link will be only effected on the starting of build, and the original link will only set to the mainpage, and after @wangxiyuan changed, will be set to the page only show related job, just like below:

image

[1] http://80.158.17.129/status/change/1,fe277e51fdd16e1731cde51e30f614f3ff36c79f

File: zuul.d/pipelines.yaml (hunk 1 above)
Comment (kiwik) [reply zuul.d/pipelines.yaml]: @Yikun got it, it's `pending` job status url, thank you @wangxiyuan and @Yikun , LGTM

**Comment by liusheng** [comment]
thanks

**Pull request merged**